lmctl/drivers/lm/etsipkgmgmtdriver.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import requests
+import json
+from .base import LmDriver, NotFoundException
+
+logger = logging.getLogger(__name__)
+
+class EtsiPackageMgmtDriver(LmDriver):
+    """
+    Client for managing packages
+    """
+
+    def __init__(self, lm_base, lm_security_ctrl=None):
+        super().__init__(lm_base, lm_security_ctrl)
+
+    def __packages_api(self):
+        return '{0}/api/etsi/vnfpkgm/v2/vnf_packages'.format(self.lm_base)
```

Comment:
To ensure lmctl can work on many platforms it's best we don't put forward slashes in path strings and instead use something like `os.path.join` to combine directories into a path.